tarpc/src/macros.rs

```diff
@@ -569,6 +581,20 @@ mod functional_test {
         assert_eq!(3, client2.add(1, 2).get().unwrap());
     }
 
+    #[test]
+    fn async_try_clone_unix() {
+        let handle = Server.spawn_with_config(UnixTransport("/tmp/test"),
+                                              Config::default()).unwrap();
+        let client1 = AsyncClient::with_config(handle.dialer(),
+                                               Config::default()).unwrap();
+        let client2 = client1.try_clone().unwrap();
+        assert_eq!(3, client1.add(1, 2).get().unwrap());
+        assert_eq!(3, client2.add(1, 2).get().unwrap());
+        drop(client1);
```

Comment:
If we don't call `shutdown` the server won't shut down. I guess that's not a big deal since it'll shut down when the process exits? Will change preemptively...